tests/TestCase/Utility/HashTest.php

```diff
@@ -506,6 +506,13 @@ public function testMaxDimensions()
         ];
         $result = Hash::maxDimensions($data);
         $this->assertEquals($result, 5);
+
+        $data = [
+           '1' => false,
+           '2' => ['2.1' => '2.1.1']
+        ];
+        $result = Hash::maxDimensions($data);
+        $this->assertEquals($result, 2);
```

Comment:
That would be great. You can push an additional commit to the existing branch to update the pull request.